.github/pull_request_template.md

```diff
@@ -41,5 +41,12 @@ this issue is not fixed.
 
 How can we validate this issue is fixed? Please provide the steps and the prove this issue is fixed.
 
+
+## Docs
+
+If docs need to be updated, please add a link to a PR to https://github.com/SUSE/doc-caasp.
```

Comment:
```suggestion
If docs need to be updated, please create an issue in https://github.com/SUSE/doc-caasp and link it in this PR.
```

Reply:
If we ask the developers to create an issue ... they won't create a PR. The idea is that the developer provides the docs, as they have been doing, as PR for you to review. If they create only the issue, then you have to create the docs and, given we have more devs than writers, this will be a bottleneck.

Reply:
Right. I didn't understand that at first, but it makes sense. It will just be up to the documentarian to then create an issue, so that we can allot time for reviewing or otherwise editing the PR.